vmdb/app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -31,3 +31,13 @@
               = link_to(_("%s (%s)") % [ui_lookup(:table => "containers"), num_containers],
                   {:action => 'show', :id => @record, :display => 'containers'},
                   :title => _("View %s") % ui_lookup(:table => "containers"))
+        - if role_allows(:feature => "container_service_show_list")
+          - num_services = @record.number_of(:container_services)
+          - if num_services == 0
+            %li.disabled
+              = link_to(_('Container Services (0)'), "#")
```

Comment:
This should probably use the existing `(_("Container Services (%s)") % num_services` format instead of introducing an extra string that needs internationalization.